README.md

```diff
@@ -32,6 +32,7 @@ Good luck.
 - Python
   - [Python Documentation](https://www.python.org/doc/)
   - [Flask](https://flask.palletsprojects.com/en/1.1.x/)
+   -[How to Create Python Virtual Environments on Ubuntu 18.04](https://linuxize.com/post/how-to-create-python-virtual-environments-on-ubuntu-18-04/)
```

Comment:
@hema-prudvi the indentation of the line 35 is not align with line 34

Reply:
fixed indentation

Reply:
Thank you hema for fixing the indentation , and welcome more contributions from you